packages/web/src/components/comments/CommentForm.tsx

```diff
@@ -39,6 +33,8 @@ type CommentFormProps = {
   isEdit?: boolean
 }
 
+// TODO: KJ - There is a bit of hacky-ness here with the input. It does not actually work with the form so the form can probably be taken out
```

Comment:
🤔 thats annoying; I'd be curious why it doesnt work with a form but its not really much of an issue since we can easily work around it

Reply:
the useField hook from formik passes some props to the textarea normally and controls the value of the input, but that doesn't work well with the current setup of the composer input. It could probably be updated to work better with the form, but I didn't want to spend too much time on it

Reply:
Ah yeah that makes sense. No I agree that its totally fine to ignore; the only real reason I included a form here was for accessibility and out of the box enter->submit keyboard behavior. Easy to make sure we solve for those without a form tho